public/openapi-3-v3.1.json

```diff
@@ -489,6 +493,9 @@
               }
             }
           },
+          "400": {
+            "$ref": "#/components/responses/BadRequest"
+          },
           "404": {
```

Comment:
here can be the one line version to have it same as is on other places
`"$ref": "#/components/responses/NotFound"`

I noticed that the "more lines version" is several times in document ... I think it is not wrong but maybe we can have it in same way for all occurrences

Reply:
Right, I must have missed those. Fixed!